### ISSUE TITLE
QQ syntax

### ISSUE THREAD

**Issue by jkarni** [alpmestan/servant#15]
How should the `ReqBody` param be specified in the QuasiQuotes syntax?

Here's one idea:

```
GET    | hello/name:Text/capital:Bool   | Greet |
POST   | greet                          | Greet | Greet
DELETE | delete/greetid:Text            |       |
```

The first column indicates what the `ReqBody` is, the second what the parameter to `Get`, `Post`, etc. is

Another idea is to use haskell-like type syntax:

```
GET     hello/name:Text/capital:Bool    Greet
POST    greet                           Greet -> Greet
DELETE  delete/greetid:Text             ()
```

where `a -> b` means `ReqBody` is of type `a`, and the response is of type `b`.

@alpmestan @soenkehahn

**Comment by alpmestan** [comment]
I quite like the alternative idea actually :) Am I the only one?

**Comment by jkarni** [comment]
Closed by #16
